packages/health-client/src/health-checker.ts

```diff
@@ -19,7 +20,8 @@ type Argv = {
 };
 
 const argv: Argv = yargs.argv as any;
-const cred = new A11yServiceCredential(argv.clientId, argv.clientSecret, argv.clientId, argv.authorityUrl);
+const logger = new GlobalLogger([new ConsoleLoggerClient(undefined, console)], process);
+const cred = new A11yServiceCredential(argv.clientId, argv.clientSecret, argv.clientId, argv.authorityUrl, logger);
```

Comment:
why are we passing undefined? also why not use dependency injection container like in other places?

Reply:
This is the script we use for E2E validation in the release pipeline, which doesn't have a dependency injection container. The undefined value is for the ServiceConfiguration. Since we're only using one configuration value to enable/disable logging, and we don't use the service config anywhere else in the script, it seemed like overkill to create one just for this.

Reply:
why not move all this inside the async function.
& create other instances after logger.setup is called.
& use logger for logging instead of console.log in this file